src/logic/operate.js

```diff
@@ -0,0 +1,26 @@
+import BigJs from 'big.js/index';
+
+
+const operate = (numerOne, numberTwo, operation) => {
+  let result = null;
+  switch (operation) {
+    case '+':
+      result = BigJs(numerOne) + BigJs(numberTwo);
+      break;
+    case '-':
+      result = BigJs(numerOne) - BigJs(numberTwo);
+      break;
+    case '*':
+      result = BigJs(numerOne) * BigJs(numberTwo);
+      break;
+    case '÷':
+      result = BigJs(numerOne) / BigJs(numberTwo);
+      break;
```

Comment:
dont forget to manage division by `ZERO` on milestone 5, else it will throw an error...we'll be looking forward to that